leptos_reactive/src/spawn_microtask.rs

```diff
@@ -14,11 +14,13 @@ cfg_if! {
         }
 
         #[cfg(any(feature = "csr", feature = "hydrate"))]
-        #[wasm_bindgen::prelude::wasm_bindgen(
-            inline_js = "export function microtask(f) { queueMicrotask(f); }"
-        )]
-        extern "C" {
-            fn microtask(task: wasm_bindgen::JsValue);
+        fn microtask(task: wasm_bindgen::JsValue) {
+            use js_sys::{Reflect, Function};
+            use wasm_bindgen::prelude::*;
+            let window = web_sys::window().expect("window not available");
+            let queue_microtask = Reflect::get(&window, &JsValue::from_str("queueMicrotask")).expect("queueMicrotask not available");
+            let queue_microtask = queue_microtask.dyn_into::<Function>().expect("queueMicrotask not a function");
```

Comment:
This should probably just used `.unchecked_into()` rather than `.dyn_into()` and unwrapping.